src/rgen/Microsoft.Macios.Generator/Emitters/InterfaceEmitter.cs

```diff
@@ -10,6 +11,7 @@ namespace Microsoft.Macios.Generator.Emitters;
 class InterfaceEmitter (ISymbolBindingContext<InterfaceDeclarationSyntax> context, TabbedStringBuilder builder) : ICodeEmitter {
 #pragma warning restore CS9113 // Parameter is unread.
 	public string SymbolName { get; } = string.Empty;
+	public IEnumerable<string> UsingStatements => [];
```

Comment:
Same - allegedly this is also cached.

Reply:
I believe that the IL of both is going to be the same or similar:
```csharp
using System;
using System.Linq;
using System.Collections.Generic;

namespace Test;

public class ExtensionFilter 
{
    IEnumerable<String> Test = Enumerable.Empty<String>();
    IEnumerable<String> Test2 = [];
}
```
That generates the following:
```
.method public hidebysig specialname rtspecialname 
        instance void .ctor () cil managed 
    {
        // Method begins at RVA 0x2050
        // Code size 30 (0x1e)
        .maxstack 8

        IL_0000: ldarg.0
        IL_0001: call class [System.Runtime]System.Collections.Generic.IEnumerable`1<!!0> [System.Linq]System.Linq.Enumerable::Empty<string>()
        IL_0006: stfld class [System.Runtime]System.Collections.Generic.IEnumerable`1<string> Test.ExtensionFilter::Test
        IL_000b: ldarg.0
        IL_000c: call !!0[] [System.Runtime]System.Array::Empty<string>()
        IL_0011: stfld class [System.Runtime]System.Collections.Generic.IEnumerable`1<string> Test.ExtensionFilter::Test2
        IL_0016: ldarg.0
        IL_0017: call instance void [System.Runtime]System.Object::.ctor()
        IL_001c: nop
        IL_001d: ret
    } // end of method ExtensionFilter::.ctor
```
So we are between Array.Empty and the Linq version.